web-build/ioncube/download-loaders.sh

```diff
@@ -1,9 +1,10 @@
 #!/bin/bash
 #ddev-generated
 set -e
-ARCH=$(/usr/bin/arch)
 
-if [ "$ARCH" = "arm64" ]; then
+ARCH="$(/usr/bin/arch)"
```

Comment:
This is where I suggested using `uname -m` instead.

```suggestion
ARCH="$(uname -m)"
```